index.html

```diff
@@ -3089,6 +3095,98 @@ <h2>
 function specified here.
         </p>
 
+        <section>
+            <h3>
+DID Resolution Input Metadata Properties
+            </h3>
+            
+            <p>
+The possible properties within this structure and their possible values are defined by [[DID-SPEC-REGISTRIES]].
+This specification defines the following common properties.
+            </p>
+            
+            <dl>
+                <dt>
+accept
+                </dt>
+                <dd>
+The MIME type of the caller's preferred representation of the <a>DID document</a>. The <a>DID resolver</a> implementation 
+MAY use this value to determine the representation of the returned <code>did-document</code> if such
+a representation is supported and available. This property is OPTIONAL.
+                </dd>
+            </dl>
+        </section>
+
+        <section>
+            <h3>
+DID Resolution Metadata Properties
+            </h3>
+            
+            <p>
+The possible properties within this structure and their possible values are defined by [[DID-SPEC-REGISTRIES]].
+This specification defines the following common properties.
+            </p>
+
+            <dl>
+                <dt>
+content-type
+                </dt>
+                <dd>
+The mime-type of the returned <code>did-document</code>.
+If the resolution is successful, and if the <code>did-document</code> is returned in the form of a byte stream, this property is REQUIRED.
```

Comment:
I'm not sure I understand what byte stream means and it seems to be the important nuance of this discussion. The one thing I understand is that because we've got an abstract model we can't assume it will always be a JSON object because it could be CBOR as well or some other form and a byte stream is the least common denominator of them all. 

However, this language leaves me confused as to what I would need to do to practically implement around this. Could someone clarify which type in a programming language this is referring to? For example, in typescript is this mean I should expect a Uint8Array in and need to encode it using a CBOR to JSON converter if I want to move it around in memory as JSON? 

What if I expect the did document to be JSON(-LD)? Should I expect to utf8 encode the bytes first and then JSON.parse() the string I just got?

As the language currently stands in this I'm left a bit confused as to these types of implementation considerations, so I'd like some further clarification. With that in mind, I recognize some of this has to stay relatively vague to make it compatible with an abstract data model.